webapp/packages/core/src/app/shared/NodesManager/NodesManagerService.ts

```diff
@@ -80,6 +80,13 @@ export class NodesManagerService {
     });
   }
 
+  async refreshNode(nodeId: string) {
+    console.log('refresh', nodeId);
+    await this.purgeServerCache(nodeId);
+    this.nodesStore.removeNode(nodeId);
+    await this.updateNodeInfo(nodeId);
```

Comment:
remove can cause many bugs in case of an update, so we probably must check the result of request inside `updateNodeInfo` and remove node if it does not exist anymore, on other cases we probably just update node related data

Reply:
to be done later